plugins/woocommerce-admin/client/products/sections/pricing-section.tsx

```diff
@@ -56,6 +75,51 @@ export const PricingSection: React.FC = () => {
 		'woocommerce'
 	);
 
+	const { dateFormat, timeFormat } = useSelect( ( select ) => {
+		const { getOption } = select( OPTIONS_STORE_NAME );
+		return {
+			dateFormat: ( getOption( 'date_format' ) as string ) || 'F j, Y',
+			timeFormat: ( getOption( 'time_format' ) as string ) || 'H:i',
+		};
+	} );
+
+	const onSaleScheduleToggleChange = ( value: boolean ) => {
```

Comment:
Can we record when the toggle button is turned on/off?

Reply:
Good idea! Done in 390b41a.